modules/kmm-day1-kernel-module-image.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT 
+[id="kmm-day1-kernel-module-image_{context}"]
+= The kernel module image
+
+The Day 1 functionality uses the same DTK based image leveraged by Day 2 KMM builds. OOT kernel module should be located under `/opt/lib/modules/${kernelVersion}`.
```

Comment:
Can we add a link to Driver ToolKit docs here? https://docs.openshift.com/container-platform/4.13/hardware_enablement/psap-driver-toolkit.html
Also I would avoid "OOT" and use "out-of-tree" here.

Reply:
Changed. Added link.